openwisp_monitoring/monitoring/tests/__init__.py

```diff
@@ -53,21 +53,21 @@ def _create_object_metric(self, **kwargs):
             kwargs['is_healthy'] = True  # backward compatibility with old tests
         return self._create_general_metric(**opts)
 
-    def _create_threshold(self, **kwargs):
-        t = Threshold(**kwargs)
-        t.full_clean()
-        t.save()
-        return t
+    def _create_alert_threshold(self, **kwargs):
+        alert_s = AlertThreshold(**kwargs)
```

Comment:
I understand alert_s comes from AlertSettings. Since we are not using that name anymore I think `t` or `at` are more meaningful.